modules/extension/tile-client/src/main/java/org/geotools/tile/TileService.java

```diff
@@ -349,6 +352,7 @@ public BufferedImage loadImageTileImage(Tile tile) throws IOException {
      * <p>Subclasses services (such as WMTS) may have some more hints about the tile TTL, so a more
      * controllable cache should be implemented in these cases.
      */
+    @Deprecated
```

Comment:
I think we can probably skip deprecation for a protected method - no one we don't know about is likely to be using it?